app/helpers/host_initiator_helper/textual_summary.rb

```diff
@@ -0,0 +1,51 @@
+module HostInitiatorHelper::TextualSummary
+  include TextualMixins::TextualGroupTags
+
+  #
+  # Groups
+  #
+
+  def textual_group_properties
+    TextualGroup.new(
+      _("Properties"),
+      %i[
+        name
+        ems
+        physical_storage
+      ]
+    )
+  end
+
+  def textual_group_san_addresses
+    san_addresses_values = []
+
+    san_addresses = SanAddress.where(owner_id: @record.id)
+    san_addresses.each do |san_address|
+      san_addresses_values << san_address.get_address_info
+    end
+
+    TextualMultilabel.new(
+      _("SAN Addresses"),
+      :labels => [_("Type"), _("Value")],
+      :values => san_addresses_values
+    )
+  end
+
+  #
+  # Items
+  #
+
+  def textual_name
+    {:label => _("Name"), :value => @record.name}
+  end
+
+  def textual_ems
+    textual_link(@record.ext_management_system)
+  end
+
+  def textual_physical_storage
+    storage_id = @record.physical_storage_id
```

Comment:
You can just go `textual_link(@record.physical_storage)`, since you had defined `belongs_to :physical_storage` on the host initiatro.

Reply:
agree. done